tools/src/linter/components/base/FileValidator.ts

```diff
@@ -22,10 +22,10 @@ export default class FileValidator extends ValidatorBase {
   }
 
   validate (): ValidationError[] {
-    const extension_error = this.validate_extension()
-    if (extension_error) return [extension_error]
     const yaml_error = this.validate_yaml()
     if (yaml_error) return [yaml_error]
+    const extension_error = this.validate_extension()
+    if (extension_error) return [extension_error]
```

Comment:
Is there a reason for this reordering? a `txt` file should result in a failure before we try to parse its content as YAML right?

Reply:
Yes, the yaml file will fail to parse with eslint, and when you ignore it in the configuration it will still try to load it and fail, so I needed to rename it. 

But after renaming it the linter test cannot test failing yaml parsing anymore because it trips on the extension. This seems like the easiest solution.